public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -593,6 +597,24 @@ export class ElasticDatasource
           bucketAggs,
         };
 
+      case SupplementaryQueryType.LogsSample:
+        isQuerySuitable = !!(
+          query.metrics?.length === 1 &&
```

Comment:
We should show logs sample only for `date_histogram` queries, which is a type of metric queries.